system_test/aest_oracles_SUITE.erl

```diff
@@ -309,18 +309,25 @@ pipelined_query_test(Opts, Cfg) ->
     start_node(node2, Cfg),
     wait_for_startup([node1, node2], 4, Cfg),
 
-    %% Give some tokens to the oracle account
-    post_spend_tx(node1, ?MIKE, OAccount, 1, #{ amount => 600000 }),
+    %% Give tokens away
+    GiveAwayAmount = 600000,
+    aest_nodes:wait_for_value({balance, MPubKey, 2*GiveAwayAmount}, [node1], 10000, []),
```

Comment:
I see all nodes in test have Mike as beneficiary, so idea is ensuring that account is present in chain so that more relaxed check on nonce is applied. Ok.

Reply:
(I read https://github.com/aeternity/epoch/pull/1763#issue-230054172 for idea of more relaxed check.)